docs/quantum_research/solver_properties_specific.rst

```diff
@@ -139,7 +139,7 @@ Annealing Schedule
 ------------------
 
 Download the annealing schedule for the QPU here:
-:download:`Advantage2_system1.1 Excel spreadsheet <../downloadables/09-1312A-A_Advantage2_system1_1_annealing_schedule.xlsx>`.
+:download:`Advantage2_system1.2 Excel spreadsheet <../downloadables/09-1312A-A_Advantage2_system1_2_annealing_schedule.xlsx>`.
```

Comment:
The PN and filename have not been updated but they should be and then this link should be updated too